lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -85,7 +87,8 @@
                           final ClasspathProvider classpath,
                           final YarnProxyUser yarnProxyUser,
                           final SecurityTokenProvider tokenProvider,
-                          final YarnSubmissionParametersFileGenerator jobSubmissionParametersGenerator) {
+                          final YarnSubmissionParametersFileGenerator jobSubmissionParametersGenerator,
+                          @Parameter(NodeLabelExpression.class) final String nodeLabelExpression) {
```

Comment:
How do you pass the value of nodeLabelExpression to YarnJobSubmissionClient and where do you bind the value to this named parameter?